packages/xo-server/src/xo-mixins/remotes.js

```diff
@@ -84,6 +88,22 @@ export default class {
     return handler.test()
   }
 
+  async getAllRemotesInfo() {
```

Comment:
This should return the available info for each remotes, and should not contain the remotes themselves, those are obtained through `getAllRemotes`.